sklearn/utils/_unittest_backport.py

```diff
@@ -0,0 +1,224 @@
+"""
+This is a backport of assertRaises() and assertRaisesRegex from Python 3.5.4
```

Comment:
Why has this been added?

Reply:
This is my fault, I pulled from origin and messed things up... Tha's why I tried reverting to the commit before all of this happened

Reply:
I removed the files now, sorry for the mess